### PR TITLE
Homework for lesson 1

### PR DESCRIPTION
**Pull request by OKrylov**

### CODE REVIEW

File: src/java1/lesson1/homework/HomeworkApp1.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+
+    private static void printThreeWords() {
+        System.out.println("Orange");
```
Comment (OKrylov) [inline src/java1/lesson1/homework/HomeworkApp1.java:14]: test comment

File: src/java1/lesson1/homework/HomeworkApp1.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+    private static void checkSumSign() {
+        int a = 3;
+        int b = -17;
+        if (a + b >= 0) {
+            System.out.println("Сумма положительная");
```
Comment (OKrylov) [inline src/java1/lesson1/homework/HomeworkApp1.java:23]: Test comment 2

File: src/java1/lesson1/homework/HomeworkApp1.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package java1.lesson1.homework;
```
Comment (DimaMinin) [inline src/java1/lesson1/homework/HomeworkApp1.java:1]: +

**Review by DimaMinin — APPROVED** [review]

**Review by AlexeySenkin — APPROVED** [review]

**Review by stadiumarcadium — APPROVED** [review]
GJ

**Review by sluKing — APPROVED** [review]

**Review by Darrasul — CHANGES_REQUESTED** [review]
test

**Review by MariaBigCatKuzmina — CHANGES_REQUESTED** [review]
тест Request changes

**Review by DVPesh — CHANGES_REQUESTED** [review]
Тест

**Pull request merged**